run-tests-with-build-business-central-community/pom.xml

```diff
@@ -112,30 +113,26 @@
                             </goals>
                             <pomIncludes>
                                 <pomInclude>kie-wb-common/kie-wb-common-services/kie-wb-common-compiler/kie-wb-common-compiler-distribution/pom.xml</pomInclude>
+                                <pomInclude>kie-wb-common/kie-wb-common-screens/kie-wb-common-default-editor/kie-wb-common-default-editor-api/pom.xml</pomInclude>
                             </pomIncludes>
                         </configuration>
                     </execution>
                     <execution>
-                        <id>run-tests</id>
-                        <phase>test</phase>
-                        <goals>
-                            <goal>run</goal>
-                        </goals>
                         <configuration>
                             <pomIncludes>
-                                <pomInclude>drools-wb/*/pom.xml</pomInclude>
-                                <pomInclude>jbpm-wb/*/pom.xml</pomInclude>
+                                <pomInclude>drools.wb/*/pom.xml</pomInclude>
```

Comment:
where are modules with such name? I suppose `.` is not any wildcard, is it?

Reply:
The problem is that the directory structure in pom.xml was setted to drools.wb and jpbm.wb (see e.g. [1]). I fixed it wrongly, I'm going to fix it properly, thanks.

[1] https://github.com/kiegroup/run-tests-with-build/blob/faba95cd5ca1da032ca4b122085e9783a192b0fc/run-tests-with-build-business-central-community/pom.xml#L28

Reply:
fixed the target directory names (from drools/jbpm.wb to drools/jbpm-wb) in business-central module